database.sql

```diff
@@ -0,0 +1,54 @@
+-- basic table set up
+
+CREATE DATABASE beer_db
+
+
+CREATE TABLE member (
+  id serial,
+  name varchar(50) NOT NULL,
+  PRIMARY KEY (id)
+);
+
+  CREATE TABLE event (
+    id serial,
+    name varchar(50) NOT NULL UNIQUE,
+    PRIMARY KEY (id)
+  );
+
+  CREATE TABLE member_map (
+  id serial,
+  member_id INT NOT NULL,
+  event_id INT NOT NULL,
+  PRIMARY KEY (id),
+  FOREIGN KEY (member_id) REFERENCES member (id),
+  FOREIGN KEY (event_id) REFERENCES event (id)
+
+);
+
+CREATE TABLE suggestions (
```

Comment:
should be singular 'suggestion' for table name

Reply:
Will correct